### ISSUE TITLE
Empty "Most Views" section 

### ISSUE THREAD

**Issue by EnzoVezzaro** [oceanprotocol/market#1885]
**Describe the bug**
The "Most Views" section is empty. 

<img width="1173" alt="Screenshot 2023-02-21 at 11 31 21" src="https://user-images.githubusercontent.com/13741335/220388318-9874d119-a98d-4874-8e3b-44704934e8db.png">

**Comment by mihaisc** [comment]
yep, this is because the market was moved to vercel. I'll switch it back to netlify soon

**Comment by jamiehewitt15** [comment]
Yeah, it would be good to fix this or just comment it out until the market is back on netlify

**Comment by mihaisc** [comment]
it was commented
